spec/decisions/simple_decision_tree_spec.rb

```diff
@@ -139,6 +139,8 @@
   it_behaves_like 'a generic decision', :solicitor_declaration, :claim_confirmation, Steps::SolicitorDeclarationForm,
                   action_name: :show
 
+  it_behaves_like 'a generic decision', :supporting_evidence, :check_answers, Steps::SupportingEvidenceForm, action_name: :show
+
```

Comment:
prefer to have in order of pages as it makes finding the test slightly easier. :)